homeassistant/components/light/template.py

```diff
@@ -210,6 +205,7 @@ def async_turn_off(self, **kwargs):
     @asyncio.coroutine
     def async_update(self):
         """Update the state from the template."""
+        print("ASYNC UPDATE")
```

Comment:
@pvizeli Can this debug statement be removed?

Reply:
Probably just a stale leftover from testing. Make a PR and remove it. :+1: